src/hooks/useSearchParams.tsx

```diff
@@ -25,12 +25,18 @@ export const useSearchParams = (): [MappedSearchParams, SetURLSearchParams] => {
   const [searchParams, internalSetSearchParams] = React.useState(toMap(location.search));
 
   /**
-   * Update and add to the existing search parameters on the URL and the internal state.
+   * Change existing search parameters on the URL and the internal state.
+   * Operations:
+   * 1. ADD - by adding a new key:value pair to the object
+   * 2. UPDATE - provide a new value for an exisitng key
+   * 3. DELETE - set value 'undefined' for an existing key
```

Comment:
Good thoughts, but not what's implemented right?

Reply:
The intention was to describe the current state (well ...with the addition below). I'll try to re-phrase.

Reply:
"Merge new search parameter into existing parameters.  Any key with an `undefined` value will be removed."